src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/EndOfServiceReportControllerTest.kt

```diff
@@ -93,11 +97,12 @@ class EndOfServiceReportControllerTest {
 
   @Test
   fun `update end of service report with no extra outcome`() {
+    val referral = referralFactory.createSent()
     val endOfServiceReportId = UUID.randomUUID()
     val outcome = CreateEndOfServiceReportOutcomeDTO(UUID.randomUUID(), AchievementLevel.ACHIEVED, null, null)
     val updateEndOfServiceReportDTO = UpdateEndOfServiceReportDTO("info", outcome)
 
-    val endOfServiceReport = SampleData.sampleEndOfServiceReport()
+    val endOfServiceReport = SampleData.sampleEndOfServiceReport(referral = referral)
```

Comment:
In one of my current pr's I've update the EndOfServiceReport & outcome factories, so we could update this to use those factories at some point. Not necessarily now.

https://github.com/ministryofjustice/hmpps-interventions-service/pull/200/files

Reply:
Just seen you have also created them later on in this pr so can ignore above